examples/multimodal_v1/components/direct_processor.py

```
@@ -0,0 +1,215 @@
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+import uuid
+from enum import Enum
+from typing import AsyncIterator, Tuple, Union
+import torch
```

Ruff (0.12.2)

`torch` imported but unused

Remove unused import: `torch`

(F401)